futurenhs.app/form-configs/content-block-text.ts

```diff
@@ -30,9 +27,10 @@ export const contentBlockTextForm: FormConfig = {
                     name: 'mainText',
                     text: {
                         label: 'Main text',
+                        hint: 'This could be an introduction to your group, remember to include key words this group can be found in search'
```

Comment:
'remember to include key words _so that_ this group can be found in search' ??

Reply:
Good spot - resolved